ffi/capi/dart/package/LICENSE

```diff
@@ -0,0 +1 @@
+--license TBD--
```

Comment:
What should the license be? @robertbastian @Manishearth

Reply:
copy the license from the root folder